Metro-Records/la-metro-dashboard
# Update production database url after councilmatic db upgrade

## Overview

See title.

Handles #127 
Connected to Metro-Records/la-metro-councilmatic#1065

### Checklist


## Testing Instructions

 * Run `blackbox_postdeploy`
 * Verify that the `LA_METRO_DATABASE_URL` variable has been updated in `configs/production/.env` (it should match the Heroku env var `DATABASE_URL` in the `la-metro-councilmatic-prod` app)
